src/main/resources/changelog/scripts/v-1.0.0/resource_graph/migration/resources_date_fields.sql

```diff
@@ -0,0 +1,12 @@
+alter table resources
+  add column created_date timestamp default current_timestamp not null,
+  add column updated_date timestamp default current_timestamp not null,
```

Comment:
is `updated_date` not null? I think updated_date will be null for new resources. updated_date will be populated when the resource is first updated. Can we do that?

Reply:
as per the story requirements when a cataloger saves a Work resource for the first time, the created_date and updated_date fields are auto-populated with the same value